### PR TITLE
Display rules.abaplint.org for check and individual messages

### LINKED ISSUE #40: AbapLint Backend not showing version
The backend does not show it's abap-lint version in the log or in the SCI results.

Both would be very helpfull with the handling of tickets.

### LINKED ISSUE #114: ADT: issue display
image

Compared to e.g. abapOpenChecks:

image

### PR DESCRIPTION
**Pull request by mbtools**
Clicking on "i" icon for abaplint root and categories will display "rules.abaplint.org"

image

Clicking on "i" icon for individual messages will display "rules.abaplint.org/...rule..." 

image

Mapping is done via /api/v1/list_rules
https://github.com/mbtools/abaplint-sci-server/blob/master/src/api/list_rules.ts

Also fixes the display of rule description in ATC and ADT:

image

Closes #114 and https://github.com/abaplint/abaplint-sci-server/issues/21

Also adds abaplint version to output as info msg: 

image

Closes #40

### CODE REVIEW

File: src/zcl_abaplint_check.clas.abap [hunk 1]
```diff
@@ -92,10 +141,35 @@ CLASS ZCL_ABAPLINT_CHECK IMPLEMENTATION.
...
+
+  ENDMETHOD.
+
+
   METHOD get_message_text.
 
     IF p_code = c_no_config.
       p_text = 'No configuration found when looking at package hierarchy, &1'.
+    ELSEIF p_code CP 'LINT_*'.
+      READ TABLE scimessages INTO smsg TRANSPORTING text
```
Comment (larshp) [inline src/zcl_abaplint_check.clas.abap:166]: is `smsg` used for something? if not then I suggest adding a new local variable

**Comment by mbtools** [comment]
the other issue requires an update of abaplint/deps.

**Comment by larshp** [comment]
updated, https://github.com/abaplint/deps/commit/f0cfa8c50ed1b6c6ee9dcb7d3d994c2f9f22908c

try pushing another commit

**Comment by larshp** [comment]
`run_end` added in https://github.com/abaplint/deps/commit/0f55a6b9362c0803bd60ace21642bda6867a2a6c

**Comment by mbtools** [comment]
Yay! Thanks. That's it for this one

**Comment by larshp** [comment]
cool 👍 will merge it tomorrow

**Pull request merged**